src/Resources/services.yml

```diff
@@ -125,17 +127,35 @@ services:
       - '@spl.priority.queue'
   jaeger.debug.extractor:
     alias: 'jaeger.debug.extractor.chain'
+  jaeger.name.generator.controller:
+    class: Jaeger\Symfony\Name\Generator\ControllerNameGenerator
+    tags:
+      - {name: 'kernel.event_subscriber' }
   jaeger.name.generator.default:
     class: Jaeger\Symfony\Name\Generator\DefaultNameGenerator
     tags:
       - {name: 'kernel.event_subscriber' }
       - {name: 'jaeger.name.generator', priority: -1}
+  jaeger.name.generator.request:
+    class: Jaeger\Symfony\Name\Generator\RequestNameGenerator
+    tags:
+      - {name: 'kernel.event_subscriber' }
+      - {name: 'jaeger.name.generator', priority: 1}
+  jaeger.name.generator.command:
+    class: Jaeger\Symfony\Name\Generator\CommandNameGenerator
+    tags:
+      - {name: 'kernel.event_subscriber' }
+      - {name: 'jaeger.name.generator', priority: 1}
   jaeger.name.generator.chain:
     class: Jaeger\Symfony\Name\Generator\NameGeneratorChain
     arguments:
       - '@spl.priority.queue'
   jaeger.name.generator:
     alias: 'jaeger.name.generator.chain'
+  jaeger.name.generator.shorten:
```

Comment:
Should we provide a way to disable it?
Or user should just use a high max value, if it doesn't care about operation name length?